fiftyone/utils/labelbox.py

```diff
@@ -62,6 +62,7 @@ def import_from_labelbox(
         [
             {
                 "ID": <labelbox-id>,
+                "DataRow ID": <datarow-id>,
```

Comment:
I would recommend updating the documentation like so:

```

        [
            {
                "DataRow ID": <labelbox-id>,
                "Labeled Data": <url-or-None>,
                "Label": {...},
                ...
            }
        ]
```

The point of the documentation is to help the user understand what value they should be storing in the `labelbox_id_field` of their samples